src/EnergyPlus/SolarShading.cc

```diff
@@ -8616,6 +8609,10 @@ namespace SolarShading {
                                 // Opaque surface
 
                                 AISurf(FloorNum) += BTOTWinZone * ISABSF(FloorNum) / Surface(FloorNum).Area; //[-]
+                            } else if (Construct(FlConstrNum).TypeIsAirBoundaryInteriorWindow) {
+                                    TransBeamWin = 1.0;
+                                    AbsBeamWinEQL = 0.0;
+                                    AbsBeamTotWin = 0.0;
```

Comment:
This sets the window properties in the correct place.

Reply:
OK, so it's not in the equivalent layer section anymore.